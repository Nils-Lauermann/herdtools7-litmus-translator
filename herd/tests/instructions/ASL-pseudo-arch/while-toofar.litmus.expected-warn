Warning: File "./herd/tests/instructions/ASL-pseudo-arch/while-toofar.litmus": unrolling limit exceeded at At File ./herd/tests/instructions/ASL-pseudo-arch/while-toofar.litmus,
     line 13, character 2 to line 16, character 6: while loop pruned, legal outcomes may be missing.
