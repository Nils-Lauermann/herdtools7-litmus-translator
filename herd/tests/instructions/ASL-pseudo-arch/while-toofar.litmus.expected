Test while-toofar Required
States 3
0:T0.0.s=0;
0:T0.0.s=1;
0:T0.0.s=3;
Loop Ok
Witnesses
Positive: 3 Negative: 0
Condition forall (true)
Observation while-toofar Always 3 0
Hash=c98af998cfb2205ef58ad672d8de3845

